CHANGELOG.rst

```diff
@@ -2,6 +2,120 @@
 Change log for Preview-generator
 ================================
 
+----------
+0.27 / TBD
+----------
+
+:warning: This release introduce many changes with two of the basic builders changed (image, pdf) and a new one added (raw).
+
+Features
+~~~~~~~~
+
+- deprecate Pillow and Imagemagick command line builder in favor of a new Wand builder #253
+- add Webp format support to the new Wand builder #273
+- replace the old pdf builder by a new poppler-utils based builder #192
+- add a new builder for raw using rawpy
+
+-----------------
+0.26 / 2021-11-16
+-----------------
+
+Fixed issues
+~~~~~~~~~~~~
+
+- enforce office mimetype supported by libreoffice to avoid unknown mimetype issues related to system specific configuration:  #283
+
+-----------------
+0.25 / 2021-10-19
+-----------------
+
+Fixed issues
+~~~~~~~~~~~~~
+
+- fix issue with inkscape builder as required:  #263
+
+-----------------
+0.24 / 2021-10-18
+-----------------
+
+Fixed issues
+~~~~~~~~~~~~
+
+- add compatibility with inkscape v1: #263
+
+-------------------
+0.23 / 2021-06-29
+-------------------
+
+Features
+~~~~~~~~
+
+- experimental gltf support : cc69ad08ef67517b04afe62d6ed58b2eb00ce9b8
+- builder have now a weight permitting to know which builder will be used if conflict exist: #237
```

Comment:
```suggestion
- experimental GLTF support: cc69ad08ef67517b04afe62d6ed58b2eb00ce9b8
- builders have now a weight permitting to know which builder will be used in case of conflict: #237
```

Reply:
@inkhey in the list of features you have this: `deprecate Pillow and Imagemagick command line builders in favor of a new Wand builder #253`

Deprecating a builder is not a feature! You should reword your sentence, something like "new Wand-based builder #253 (which by the way deprecates Pillow and Imagemagick command line builders)

Reply:
Same remark for `replace the old PDF builder by a new poppler-utils based builder #192` The feature is "new poppler-utils based builder (which replace old PDF builder)"

Reply:
Thanks for the advice, i changed the text.